Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -59,9 +59,9 @@ def bump_cask_pr_args
   def bump_cask_pr
     args = bump_cask_pr_args.parse
 
-    # This will be run by `brew style` later so run it first to not start
-    # spamming during normal output.
-    Homebrew.install_bundler_gems!
+    # This will be run by `brew audit` or `brew style` later so run it first to
+    # not start spamming during normal output.
+    Homebrew.install_bundler_gems! if args.no_audit? && args.no_style?
```

Comment:
Looks like this should be `if !args.no_audit? || !args.no_style?`

Reply:
@Bo98 thanks: https://github.com/Homebrew/brew/pull/15703